kolibri/core/content/utils/channel_import.py

```diff
@@ -115,9 +117,19 @@ def __init__(self, channel_id, channel_version=None, cancel_check=None):
 
         # Explicitly set the destination schema version to our latest published schema version
         # Not the current schema of the DB, as we do our mapping to the published versions.
-        self.destination = Bridge(
-            schema_version=CONTENT_SCHEMA_VERSION, app_name=CONTENT_APP_NAME
-        )
+        if destination is None:
+            # If no destination is set then we are targeting the default database
+            self.destination = Bridge(
+                schema_version=CONTENT_SCHEMA_VERSION, app_name=CONTENT_APP_NAME
+            )
+        else:
+            # If a destination is set then pass that explicitly. At the moment, this only supports
+            # importing to an arbitrary SQLite file path.
```

Comment:
Assuming we don't need any SQLite specific code checks?

Reply:
We might want to add some extra guards to ensure that this is only a SQLite file path, but this is more of a proof of concept at this point, as there is no practical way to access this code path.